ct.yaml

```diff
@@ -11,3 +11,6 @@ helm-extra-args: --timeout 600s
 excluded-charts:
   # If not running on GCE, will error: "Failed to get GCE config"
   - prometheus-to-sd
+additional-commands:
+  - helm unittest --helm3 --strict --file tests/*.yaml {{ .Path }}
+  - helm unittest --helm3 --strict --file tests/**/*.yaml {{ .Path }}
```

Comment:
In `kube-prometheus-stack` we might want to have directories in the `tests` directory, therefore the second command is setup already. I guess it's easier to do it now rather than figure this out twice.